feat: Add Rapid Response reports plugin (#17)
mitodl/open-edx-plugins: src/ol_openedx_rapid_response_reports/README.rst

Rapid Response Reports Plugin
=============================

Overview
--------

A django app plugin for edx-platform to enable "Rapid Response Reports" functionality in "Instructor" tab.

**NOTE:**

> Since the edx-platform’s "Instructor Dashboard" does not support the plugin based tab extensions so we had to make some changes in the edx-platform itself to add ``Rapid Responses`` tab through this plugin.
> So, the ``edx-platform`` branch/tag you're using must include below commit for ``ol-openedx-rapid-response-reports`` plugin to work properly:

Reviewer: This can be a little more brief

```suggestion
We had to make some changes to edx-platform itself in order to add the ``Rapid Responses`` tab to the instructor dashboard, so the ``edx-platform`` branch/tag you're using must include this commit for the plugin to work properly:
```
